roles/openshift_logging_elasticsearch/tasks/main.yaml

```diff
@@ -450,6 +459,57 @@
     es_deploy_name: "{{ openshift_logging_elasticsearch_deployment_name }}"
   when: openshift_logging_elasticsearch_deployment_name != ""
 
+- set_fact:
+    _es_container_security_context: {'privileged': true}
```

Comment:
As we changed scc for service account to hostmount-anyuid, this claim is not necessary anymor.

Reply:
Fixed!